sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusErrorSource.java

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus;
+
+import com.azure.core.util.ExpandableStringEnum;
+
+import java.io.Serializable;
+
+/**
+ * Represent the operation this sdk was performing when the error happened.
+ */
+public final class ServiceBusErrorSource extends ExpandableStringEnum<ServiceBusErrorSource> implements Serializable {
+
+    private static final long serialVersionUID = -2819764417333954922L;
```

Comment:
Is there a reason we need to implement Serializable? Is this a thing? 🤔  @JonathanGiles

Reply:
Connie is right - we shouldn't be making everything serializable by default.

Reply:
We are getting this error in spotbug at build time.
[ERROR] Class com.azure.messaging.servicebus.ServiceBusAmqpException defines non-transient non-serializable instance field errorSource [com.azure.messaging.servicebus.ServiceBusAmqpException] In ServiceBusAmqpException.java SE_BAD_FIELD

Reply:
changed it to `transient`.